### ISSUE TITLE
placeholder option does not work

### ISSUE THREAD

**Issue by andidev** [summernote/angular-summernote#90]
I can't seem to set the placeholder text with neither
```
    <summernote placeholder="asdasdasd"></summernote>
```
or
```
    vm.options = {
        placeholder: "A placeholder..."
    };

    <summernote config="vm.options"></summernote>
```

is it not supported by the angular version yet?

**Comment by gongzza** [comment]
i can see the placeholder.

did you set the controllerAs?

**Comment by andidev** [comment]
Seems to be resolved with version 7.0 so I'm closing this.

**Comment by outsideris** [comment]
Good!!
